build-system/get-dep-graph.js

```diff
@@ -82,11 +82,16 @@ exports.getFlags = function(config) {
     language_in: 'ES6',
     language_out: config.language_out || 'ES5',
     module_output_path_prefix: config.writeTo || 'out/',
+    module_resolution: 'NODE',
     externs: config.externs,
     define: config.define,
     // Turn off warning for "Unknown @define" since we use define to pass
     // args such as FORTESTING to our runner.
     jscomp_off: ['unknownDefines'],
+    // checkVars: Demote "variable foo is undeclared" errors.
+    // moduleLoad: Demote "module not found" errors to ignore missing files
+    //     in type declarations in the swg.js bundle.
+    jscomp_warning: ['checkVars', 'moduleLoad'],
```

Comment:
did we need this for both compile and check-type passes?

Reply:
Shouldn't we be consistent between both?

Reply:
@choumx ideally. i just remembered the scenario you ran into originally with multi-pass, where the compilation worked but check-types failed. was that more an inconsistency with another option?